spec/rubocop/config_loader_spec.rb

```diff
@@ -386,6 +387,14 @@
       )
     end
 
+    it 'loads configuration properly when it includes non-ascii characters ' do
+      create_file(configuration_path, ['# All these cops of mine are ❤',
```

Comment:
@mikegee I don't know what would be the rationale behind that `.force_encoding('binary')`, but I just tried it and it doesn't catch the bug.

The problem here is with the external encoding, and the only way to set that is through environment variables (`LANG`, `LC_ALL`) before running ruby, or the ruby flag `-E`. This option looked heavy-handed to me at first, but after looking at the final patch, I no longer think so.

I also tried setting [Enconding.default_external](http://ruby-doc.org/core-2.3.1/Encoding.html#method-c-default_external-3D) manually, but not only it's explicitly discouraged in the docs, but it also didn't catch the bug.